### PR TITLE
create may's blog post

### PR DESCRIPTION
**Pull request by MisterE123**
First draft of the blog post.

### CODE REVIEW

File: _posts/2022-06-05-May.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+description: >
+  This month, Minetest 5.5.1 is released, nodebox rendering gets a speedboost, we get a new mapping mod, and Minetest's first walkable vehicle. Exile and Nodecore get major improvements as well. 
+image: /static/blog/2022_May/izzyscastle.webp
+forum_topic: https://forum.minetest.net/viewtopic.php?f=18&t=28105
+tags:
+  - this_month
+---
+	First, we should celebrate a major milestone. For the first time in a long time, Minetest finally has fewer than 1000 open issues. A big thank you to the Development Team for speeding up the pace of development, and to Zughy for triaging the issues and closing the ones that could safely be closed.
+	
+  This month, Minetest 5.5.1 is released, nodebox rendering gets a speedboost, we get a new mapping mod and minetest's first walkable vehicle. Exile and Nodecore get major improvements as well.
```
Comment (rubenwardy) [inline _posts/2022-06-05-May.md:15]: Missing  after this paragraph

File: _posts/2022-06-05-May.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+author: MisterE
+contributors: ["MisterE", "Rubenwardy"]
+description: >
+  This month, Minetest 5.5.1 is released, nodebox rendering gets a speedboost, we get a new mapping mod, and Minetest's first walkable vehicle. Exile and Nodecore get major improvements as well. 
+image: /static/blog/2022_May/izzyscastle.webp
+forum_topic: https://forum.minetest.net/viewtopic.php?f=18&t=28105
+tags:
+  - this_month
+---
+	First, we should celebrate a major milestone. For the first time in a long time, Minetest finally has fewer than 1000 open issues. A big thank you to the Development Team for speeding up the pace of development, and to Zughy for triaging the issues and closing the ones that could safely be closed.
```
Comment (rubenwardy) [inline _posts/2022-06-05-May.md:13]: Shouldn't this be after the introduction paragraph?

Comment (rubenwardy) [inline _posts/2022-06-05-May.md:13]: Don't begin paragraphs with tabs, this makes markdown think its code

Also, use Rewrap Text to wrap lines, this makes it easier to do line comments

* Install Rewrap Text
* Click in a paragraph
* Press  Ctrl+Shift+P
* Select "Rewrap Comment / Text"/

File: _posts/2022-06-05-May.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	
+## Mod News <a name="mods"></a>
+
+### Xmaps
+{% include figure.html src="/static/blog/2022_May/xmaps.png" caption="Treasure Maps for Minetest" %}
+
+[xmaps](https://content.minetest.net/packages/erlehmann/xmaps/) is a new mod by [erle](https://content.minetest.net/users/erlehmann/) that adds treasure-maps to the HUD. You get them by rightclicking a mapping kit. You can mark the spot with an X by rightclicking a node. Treasure Hunt, anyone?
+
+### Steampunk Blimp
+{% include figure.html src="/static/blog/2022_May/blimp_1.jpg" caption="Exploring the Skies" %}
```
Comment (rubenwardy) [inline _posts/2022-06-05-May.md:52]: Is there a screenshot without the chat / hud?

File: _posts/2022-06-05-May.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+ 
+## Art and Builds <a name="art"></a>
+
+{% include figure.html src="/static/blog/2022_May/izzyscastle.webp" caption="Izzyb's castle on Land of Catastrophe. Truly an amazing acomplishment given the difficulty of the game." %}
+
+## Server News <a name="s-news"></a>
+
+### From Minigames Galore and A.E.S. Minigames:
+<sub>minetest.eticadigitale.org:30010</sub>
+The 3rd Blockleague tournament is planned for Friday, June 24 at 21:30 - 23:15, GMT+1.
```
Comment (rubenwardy) [inline _posts/2022-06-05-May.md:73]: Missing blank line after the address to make it a paragraph

File: _posts/2022-06-05-May.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+The event will be a direct elimination competition between teams of 3 players. The winners will get a chat prefix and statues in the lobby until the next tournament.
+There are still 3 team slots left; to find people to team up with or to register your team, join us on [matrix](https://matrix.to/#/#arcadeemulationserver:matrix.org) or [IRC](https://kiwiirc.com/nextclient/irc.freeirc.org#minigames-discuss)
+
+## In Other News <a name="o-news"></a>
+
+Minetest was represented this past month at the Svilupparty game dev conference in Bologna, Italy by Zughy, Crazyminer, _Zaizen_.
```
Comment (rubenwardy) [inline _posts/2022-06-05-May.md:79]: Photo? See MT twitter

File: _posts/2022-06-05-May.md (hunk 1 above)
Comment (MisterE123) [reply _posts/2022-06-05-May.md:15]: explain?

File: _posts/2022-06-05-May.md (hunk 3 above)
Comment (MisterE123) [reply _posts/2022-06-05-May.md:52]: we can cut the screenshot...

File: _posts/2022-06-05-May.md (hunk 1 above)
Comment (rubenwardy) [reply _posts/2022-06-05-May.md:15]: looks like GitHub swallowed that, I wrote `Missing <!-- more --> after this paragraph`

Comment (MisterE123) [reply _posts/2022-06-05-May.md:15]: so, a link to more info?

**Pull request merged**